201-web-app-aspapp-on-ase-create/README.md

```diff
@@ -0,0 +1,10 @@
+# Create an App Service Plan and an App in an App Service Environment
```

Comment:
Let's rename the folder to 201-web-app-asp-app-on-ase-create so that the 'asp' is a little bit more readable